### PR TITLE
fix build error, fix years

### PR DESCRIPTION
**Pull request by darjeeling**
* fix build error on escape 
* fix years

### CODE REVIEW

**Review by ikhoon — APPROVED** [review]
👍

**Review by minwoox — APPROVED** [review]
Thanks!

File: centraldogma/dogma.py [hunk 1]
```diff
@@ -125,11 +125,11 @@ def list_files(
         """Lists files. The user should have read permission at least.
 
         :param path_pattern: A path pattern is a variant of glob as follows. |br|
-            "/\*\*" - find all files recursively |br|
-            "\*.json" - find all JSON files recursively |br|
-            "/foo/\*.json" - find all JSON files under the directory ``/foo`` |br|
-            "/\*/foo.txt" - find all files named foo.txt at the second depth level |br|
-            "\*.json,/bar/\*.txt" - use comma to match any patterns |br|
+            "/\\*\\*" - find all files recursively |br|
```
Comment (hexoul) [inline centraldogma/dogma.py:128]: Could you share where build error occurs? Recently deployed Sphinx seems okay.
- https://line.github.io/centraldogma-python/centraldogma.html#centraldogma.dogma.Dogma
- https://github.com/line/centraldogma-python/actions/workflows/pages/pages-build-deployment

As #41 did, it was fixed before. AFAIK, there was no need to escape.

Comment (darjeeling) [reply centraldogma/dogma.py:128]: @hexoul you can detect on the testing process
```
============================================== warnings summary ==============================================
centraldogma/dogma.py:125
  /Users/dj/line/src/centraldogma-python/centraldogma/dogma.py:125: SyntaxWarning: invalid escape sequence '\*'
    """Lists files. The user should have read permission at least.

centraldogma/dogma.py:148
  /Users/dj/line/src/centraldogma-python/centraldogma/dogma.py:148: SyntaxWarning: invalid escape sequence '\*'
    """Gets files. The user should have read permission at least. The difference from

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================= 81 passed, 2 warnings in 145.57s (0:02:25) =================================
```

Comment (hexoul) [reply centraldogma/dogma.py:128]: Thanks. It didn't appear, but checked after upgrading local environment.

**Review by hexoul — APPROVED** [review]

**Pull request merged**